card/mod/01_core/chunk/include.rb

```diff
@@ -1,30 +1,44 @@
 # -*- encoding : utf-8 -*-
 
-require_dependency File.expand_path( '../reference', __FILE__ )
+require_dependency File.expand_path('../reference', __FILE__)
 
 module Card::Chunk
   class Include < Reference
     cattr_reader :options
-    @@options = ::Set.new [ :inc_name, :inc_syntax, :view, :items, :type, :size, :title, :hide, :show, :structure, :params, :variant ]
+    @@options = ::Set.new [
+      :inc_name,   # name as used in nest
+      :inc_syntax, # full nest syntax
+      :items,      # handles pipe-based recursion
+
+      # _conventional options_
+      :view,       #
+      :type,       #
+      :size,       # images only
+      :title,      #
+      :hide,       # affects optional rendering
+      :show,       # affects optional rendering
+      :structure,  # override raw_content
+      :params,     #
+      :variant     #
+    ]
     attr_reader :options
 
-    Card::Chunk.register_class self, {
-      prefix_re: '\\{\\{',
-      full_re:    /^\{\{([^\}]*)\}\}/,
-      idx_char:  '{'
-    }
+    Card::Chunk.register_class(
+      self, prefix_re: '\\{\\{',
+            full_re:    /^\{\{([^\}]*)\}\}/,
+            idx_char:  '{')
 
-    def interpret match, content
+    def interpret match, _content
```

Comment:
Assignment Branch Condition size for interpret is too high. [16.88/15]<br>Method has too many lines. [16/15]